File: app/src/androidTest/java/com/example/lfrydrych/test/utils/CucumberRunner.java
package com.example.lfrydrych.test.utils;

import cucumber.api.CucumberOptions;

@CucumberOptions(features = "features",
        glue = {"com.example.lfrydrych.test.steps"},
        plugin = {"pretty"} //"html: cucumber-html-reports",
        // "json: cucumber-html-reports/cucumber.json"}
)
public class CucumberRunner {
}

File: app/src/androidTest/java/com/example/lfrydrych/test/test/CucumberRunner.kt
package com.example.lfrydrych.test.test

import cucumber.api.CucumberOptions

@CucumberOptions(features = arrayOf("features"), glue = arrayOf("com.example.lfrydrych.test.cucumber.steps"), plugin = arrayOf("pretty"), tags = arrayOf("@test2, @test3"))
class CucumberRunner
